packages/extension-chains/src/index.ts

```diff
@@ -17,6 +17,33 @@ const definitions = new Map<string, MetadataDef>(
   CENNZNetChain.map((def) => [def.genesisHash, def])
 );
 
+export function getLatestMetaFromServer(genesisHashExpected: string) {
+  try {
+    const xmlHttp = new XMLHttpRequest();
+    //xmlHttp.open( "GET", "https://raw.githubusercontent.com/cennznet/api.js/master/packages/api/src/staticMetadata.ts", false ); // false for synchronous request
+    xmlHttp.open("GET", "https://raw.githubusercontent.com/cennznet/api.js/test1/extension-releases/metadata.json", false);
```

Comment:
part of the reason to split the files is that the fetch is cheaper over the network (less data)

Reply:
should this have a timeout?

Reply:
used two different end points to get the data.. no won't need a timeout here, its sync request, if fails.. the function will return null and will be handled properly....